src-internal/Admin/Events.php

```diff
@@ -9,43 +9,43 @@
 defined( 'ABSPATH' ) || exit;
 
 use \Automattic\WooCommerce\Admin\Features\Features;
-use \Automattic\WooCommerce\Admin\Notes\AddingAndManangingProducts;
-use \Automattic\WooCommerce\Admin\Notes\ChoosingTheme;
-use \Automattic\WooCommerce\Admin\Notes\CustomizingProductCatalog;
-use Automattic\WooCommerce\Admin\Notes\FirstDownlaodableProduct;
-use \Automattic\WooCommerce\Admin\Notes\InsightFirstProductAndPayment;
-use \Automattic\WooCommerce\Admin\Notes\MobileApp;
-use \Automattic\WooCommerce\Admin\Notes\NewSalesRecord;
-use \Automattic\WooCommerce\Admin\Notes\TrackingOptIn;
+use \Automattic\WooCommerce\Internal\Admin\Notes\AddingAndManangingProducts;
+use \Automattic\WooCommerce\Internal\Admin\Notes\ChoosingTheme;
+use \Automattic\WooCommerce\Internal\Admin\Notes\CustomizingProductCatalog;
+use Automattic\WooCommerce\Internal\Admin\Notes\FirstDownlaodableProduct;
+use \Automattic\WooCommerce\Internal\Admin\Notes\InsightFirstProductAndPayment;
+use \Automattic\WooCommerce\Internal\Admin\Notes\MobileApp;
+use \Automattic\WooCommerce\Internal\Admin\Notes\NewSalesRecord;
+use \Automattic\WooCommerce\Internal\Admin\Notes\TrackingOptIn;
 use \Automattic\WooCommerce\Admin\Notes\OnboardingEmailMarketing;
```

Comment:
Can this import be deleted, given it's not used anymore? (OnboardingEmailMarketing)

Reply:
Thank you @louwie17 for your review. I like the removal idea but since this is a pretty big PR, if it's ok for you I'll merge this PR, create an issue and tackle this in a follow-up PR.

Reply:
Yeah that's fine with me

Reply:
I just created [a PR to remove the importation](https://github.com/woocommerce/woocommerce-admin/pull/8446) of `OnboardingEmailMarketing`.